__tests__/games.test.js

```diff
@@ -98,4 +98,78 @@ afterAll(() => {
           
       });
       
+  });
+
+  describe.only('PATCH /api/reviews/review_id', () => {
+      test('202: responds with a review with it\'s votes counter altered by the correct amount', () => {
+          const inc_votes = { inc_votes: -5 }
+
+          return request(app)
+          .patch("/api/reviews/2")
+          .send(inc_votes)
+          .expect(202)
+          .then((results) => {
```

Comment:
Whilst I like your use of a 202 status code, this means that a request has been accepted but hasn't yet been processed. This isn't accurate in this case sou I would use a 200 instead